### PR TITLE
Formatting JSON response for View Dependents API

### PR DESCRIPTION
**Pull request by kathleencrawford**
## Description of change

Updating the response data returned from the View Dependents endpoint to be consistent with formatting that is expected on the front end.

Original ticket here - https://github.com/department-of-veterans-affairs/va.gov-team/issues/6185

JSON response prior to this fix: 
```
{
   "number_of_records":"2",
   "persons":[
      {
         "award_indicator":"N",
         "city_of_birth":"WASHINGTON",
         "current_relate_status":null,
         "date_of_birth":"01/01/2000",
         "date_of_death":null,
         "death_reason":null,
         "email_address":"Curt@email.com",
         "first_name":"CURT",
         "gender":null,
         "last_name":"WEBB-STER",
         "middle_name":null,
         "proof_of_dependency":"Y",
         "ptcpnt_id":"32354974",
         "related_to_vet":"N",
         "relationship":"Child",
         "ssn":"500223351",
         "ssn_verify_status":"1",
         "state_of_birth":"DC"
      }
   ],
   "return_code":"SHAR 9999",
   "return_message":"Records found"
}
```

JSON response after this fix:
```
{
   "data":{
      "id":"",
      "type":"dependents",
      "attributes":{
         "persons":[
            {
               "award_indicator":"N",
               "city_of_birth":"WASHINGTON",
               "current_relate_status":null,
               "date_of_birth":"01/01/2000",
               "date_of_death":null,
               "death_reason":null,
               "email_address":"Curt@email.com",
               "first_name":"CURT",
               "gender":null,
               "last_name":"WEBB-STER",
               "middle_name":null,
               "proof_of_dependency":"Y",
               "ptcpnt_id":"32354974",
               "related_to_vet":"N",
               "relationship":"Child",
               "ssn":"500223351",
               "ssn_verify_status":"1",
               "state_of_birth":"DC"
            }
         ]
      }
   }
}
```

## Testing done

- [x] Local

## Testing planned
- [ ] Staging

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: app/controllers/v0/dependents_applications_controller.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def create
     def show
       dependent_service = BGS::DependentService.new
       response = dependent_service.get_dependents(current_user)
-      render json: response, each_serializer: DependentsSerializer
+      render json: { data: { attributes: response } }, each_serializer: DependentsSerializer
```
Comment (omgitsbillryan) [inline app/controllers/v0/dependents_applications_controller.rb]: Can you confirm that the `DependentsSerializer ` is being used at all? `response` is just a `Hash` and it looks like you mean to iterate over `response[:persons].

This observation came from @annaswims (many thanks!)

Comment (omgitsbillryan) [reply app/controllers/v0/dependents_applications_controller.rb]: https://dsva.slack.com/archives/CQHBJ5U06/p1582725724033300

Comment (kathleencrawford) [reply app/controllers/v0/dependents_applications_controller.rb]: You were right...thank you @omgitsbillryan and @annaswims.  I've made the fix so now the serializer is being used correctly.  Please take a look when you get a chance and let me know if this is ok to merge.  Thanks!

**Comment by omgitsbillryan** [comment]
Not sure why I didn't mention this the first time around... is there no spec for this? I'd expect that changing the json response structure would have broken a spec somewhere.

**Comment by omgitsbillryan** [comment]
Is there no swagger doc either? 😱

**Comment by omgitsbillryan** [comment]
I'm not gonna block the PR since your job should have been to _update_ swagger and specs, not _create_ them. Just make sure that:

1. FE folks are aware of this change
2. You file a tech debt ticket to create specs & swagger for this endpoint

**Comment by kathleencrawford** [comment]
@omgitsbillryan this ticket is the original PR where the swagger docs and spec were created: https://github.com/department-of-veterans-affairs/vets-api/pull/3838/files

this specific PR just adds the "data/id/type, etc" to the json as shown above.

**Comment by kathleencrawford** [comment]
@omgitsbillryan thank you for pointing that out.  here is the screenshot from the updated swagger:

Screen Shot 2020-03-04 at 2 07 54 PM

**Pull request merged**